name.abuchen.portfolio/src/name/abuchen/portfolio/model/PortfolioTransaction.java

```diff
@@ -68,6 +68,9 @@ public String toString()
     @Deprecated
     /* package */transient long fees;
 
+    @Deprecated
+    /* package */transient long compensation;
```

Comment:
Nur als Hinweis: die Felder "fees" und "taxes" existierende noch in der PortfolioTransaction weil die früher mal im XML an dieser Stelle persistiert wurden (bevor ich die ```Transaction.Unit``` eingeführt habe). Um auch alte Dateien noch laden zu können, muss das Feld weiterhin existieren. Damit die aber in Zukunft nicht mehr ins XML geschrieben werden, ist das Feld als ```transient``` markiert. Das macht nur Sinn für alte Attribute.